src/world/area_trd/trd_03/9A4650.c

```diff
@@ -1,5 +1,22 @@
 #include "trd_03.h"
 
-INCLUDE_ASM(s32, "world/area_trd/trd_03/9A4650", func_80240000_9A4650);
+static ApiStatus func_80240000_9A4650(ScriptInstance* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    u32 stickX;
 
-INCLUDE_ASM(s32, "world/area_trd/trd_03/9A4650", func_8024006C_9A46BC);
+    stickX = 0;
+    if (gCollisionStatus.pushingAgainstWall == 35) {
+        stickX = ((u32)GAME_STATUS->stickX) >> 0x1F;
```

Comment:
Is this the simplest this cast can get? Maybe `stickX` is a `s32` given that it is an `s32` globally...

Reply:
missed this... it may be `stickX = GAME_STATUS->stickX < 0;` or so

Reply:
Nice, thanks @pixel-stuck - I knew I was missing something there..